### DISCUSSION TITLE
How to change tables styling ?

### DISCUSSION THREAD

**Discussion by rgarrigue** [requarks/wiki#2982] [Help / Questions]
Hi there

We're having an issue with the tables' styling : it's okay for small table, but since there's no color alternance lighter / darker, no vertical separator, it's hard to read for bigger tables. e.g.

![image](https://user-images.githubusercontent.com/1872031/106265894-fba3e180-6227-11eb-9948-78e84d1b580b.png)

Is there a markdown pre defined styling for tables like `{.contrast}` ? Or I guess I can achieve this through per-page custom CSS, but I can't find a doc about it, not sure this is possible for markdown pages (VS html pages). Otherwise what should I add in Administration / Theme / Code Injection / CSS Override ? A quick hint will save me a lot of time. 

Thanks & regards !

**Comment by vnpenguin** [comment]
I use this CSS:
```
.contents DIV.table-container TABLE TD,
.contents DIV.table-container TABLE TH {
  padding: 0 5px;
  border-left: 1px solid #606060;
  border-bottom: 1px solid #606060
}
.contents DIV.table-container TABLE TD:last-child {
  border-right: 1px solid #606060
}
.contents DIV.table-container TABLE TH {
  border-top: 1px solid #606060
}
```
![20250326_004146](https://github.com/user-attachments/assets/e1f798ab-0644-4f2d-b045-ce84d9c1adda)
